ci/scripts/e2e-source-test.sh

```diff
@@ -71,10 +71,17 @@ export MYSQL_HOST=mysql MYSQL_TCP_PORT=3306 MYSQL_PWD=123456
 sqllogictest -p 4566 -d dev './e2e_test/source/cdc/cdc.share_stream.slt'
 
 
-# kill cluster and the connector node
+# kill cluster
 cargo make kill
 echo "cluster killed "
 
+# insert into mytest database (cdc.share_stream.slt)
+mysql --protocol=tcp -u root mytest -e "INSERT INTO products
+       VALUES (default,'RisingWave','Next generation Streaming Database'),
+              (default,'Materialize','The Streaming Database You Already Know How to Use');
+       INSERT INTO orders VALUES (default, '2022-12-01 15:08:22', 'Sam', 1000.52, 110, false);"
+
+
```

Comment:
Can test other things like `DELETE`, `UPDATE`, and increase the workload as well.

Reply:
Update and I found a bug in mview exec https://github.com/risingwavelabs/risingwave/issues/13346